lib/services/garbageCollector.js

```diff
@@ -40,13 +40,13 @@ function GarbageCollector(kuzzle) {
 
     if (kuzzle.funnel.overloaded) {
       // Reschedule the task to run in 1 hour
-      this.timer = setTimeout(this.run, OneHour);
+      this.timer = setTimeout(this.run.bind(this), OneHour);
```

Comment:
Since run is a property of the `GarbageCollector` object, I'm not sure binding the context is necessary.